draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1079,15 +1079,32 @@ Decoy digests increase the size of the SD-JWT. The number of decoy digests (or w
 
 ## Unlinkability
 
-Colluding Issuer/Verifier or Verifier/Verifier pairs could link issuance/presentation
-or two presentation sessions to the same user on the basis of unique values encoded in the SD-JWT
-(Issuer signature, salts, digests, etc.).
-
-To prevent these types of linkability, various methods, including but not limited to the following ones can be used:
-
-- Use advanced cryptographic schemes, outside the scope of this specification.
-- Issue a batch of SD-JWTs to the Holder to enable the Holder to use a unique SD-JWT per Verifier. This only helps with Verifier/Verifier unlinkability.
-
+Unlinkability is a property that prevents an adversary from associating two or
+more sessions of the same user. Two types of unlinkability are often considered:
+
+- Verifier/Verifier unlinkability: Two colluding Verifiers should not be able to
+  tell that they have received presentations from the same user.
+- Issuer/Verifier unlinkability: An Issuer colluding with a Verifier should
+  not be able to tell that a user that has received a credential from the Issuer
+  has presented it to the Verifier.
```

Comment:
```suggestion
- Issuer/Verifier unlinkability: An Issuer colluding with a Verifier should
  not be able to tell to which Verifier a user has presented a credential issued from this Issuer.
```

Reply:
In the proposed text, the Verifier must collude with multiple Verifiers. I think my original definition is stronger.

Reply:
Issuer/Verifier unlinkability: 

The basic property should not include the wording "colluding with a Verifier".

Proposal: An Issuer should not be able to know to which Verifier data derived from a credential issued by the Issuer has been presented to the Verifier.

Reply:
mechanically I think there are 2 things that are trying to fit into a single sentence about "Issuer/Verifier unlinkability" here:

1. phone home / issuer's learning when a specific credential is presented.
2. tracking / fingerprinting of the subject by the issuer.

The first is mitigated partially by heard privacy.
The second requires the issuer not to assign or hide correlation capable identifiers.... and even the signature is such an identifier.

I suggest starting by just writing out the behaviors you want to see, and don't want to see... and then don't give the same name to multiple behaviors.

Reply:
The difference is effectively whether the Verifier is cooperating with the attacker or whether it is just following protocol; I'll split this up accordingly.

Reply:
hey Dan, I don't love "to tell" and the concept of the speacking machines :-)

WDYT about this

````
Issuer/Verifier unlinkability: If an Issuer and a Verifier were to collude, they should not be able to determine that a user has presented a credential from the Issuer to that Verifier.
```